source/includes/entityref/paybill/_billingSyncBatchFileAttachment.md

```diff
@@ -0,0 +1,223 @@
+# Pay & Bill - BillingSyncBatchFileAttachment
+
+File attachments which can be associated with a Billing Sync Batch.
```

Comment:
Even though there is a lot of inconsistency in the doc set right now, we prefer this to be BillingSyncBatch, the entity name.